settings.gradle

```diff
@@ -22,6 +22,20 @@ sourceControl {
 
 rootProject.name = 'airbyte'
 
+// definition for dependency resolution
+dependencyResolutionManagement {
+    repositories {
+        maven {
+            url 'https://airbyte.mycloudrepo.io/public/repositories/airbyte-public-jars/'
+        }
+    }
+    versionCatalogs {
+        libs {
+            from("io.airbyte:oss-catalog:0.36.3-alpha")
```

Comment:
should this reference the deps.toml file instead?

Reply:
Sure we can, we could either use catalog or toml file. Changed to use toml file